swan-lake/resources/featured-scenarios/build-a-data-service-in-ballerina.md

```diff
@@ -426,60 +442,74 @@ import ballerina/http;
         return getAllEmployees();
     }
     
-    isolated resource function put .(@http:Payload Employee emp) returns int|error? {
+    resource function put .(Employee emp) returns int|error? {
         return updateEmployee(emp);
     }
     
-    isolated resource function delete [int id]() returns int|error? {
+    resource function delete [int id]() returns int|error? {
         return removeEmployee(id);       
     }
 
 }
 ```
 
-## Run the service 
+## Run the service
 
-Execute the command below to run the service.
+Use the [**Run**](/learn/vs-code-extension/run-a-program/) CodeLens of the VS Code extension to build and run the service, as shown below.
 
-```
-$ bal run
-```
+![Run the service](/learn/images/featured-scenarios/build-a-data-service-in-ballerina/run-the-service.gif)
+
+>**Info:** Alternatively, you can run this service by navigating to the project root (i.e., `data_service` directory) and executing the `bal run` command. 
 
-You view the output below.
+You should see the output similar to the following.
 
 ```
 Compiling source
-        foo/data_service:0.1.0
+        featured_scenarios/build_a_data_service:0.1.0
 
 Running executable
 ```
 
->**Info:** This creates an `/employees` endpoint on port `8080`, which can 
-be accessed via a browser by visiting `http://locahost:8080/employees`.
+>**Info:** This creates an `/employees` endpoint on port `8080`, which can be accessed via a browser by visiting `http://locahost:8080/employees`.
 
 ## Try the service
 
-Invoke the defined resource method by sending the `POST` request below to `http://localhost:8080/employees` with the required data as a JSON payload.
-
-```
-$ curl -X POST http://localhost:8080/employees/ -H "Content-Type: application/json" -d "{ \"employee_id\": 6, \"first_name\": \"test\", \"last_name\": \"test\", \"email\": \"test@test.com\", \"phone\": \"882 771 110\", \"hire_date\": { \"year\": 2021, \"month\": 12, \"day\": 16 }, \"manager_id\": 1, \"job_title\": \"Sales Manager\" }"
+Use the [**Try it**](/learn/vs-code-extension/try-the-services/try-http-services/) CodeLens of the VS Code extension to invoke the defined resource method by sending a `POST` request to `http://localhost:8080/employees` with the required data as shown in the JSON payload below.
+
+```json
+{
+  "first_name": "string",
+  "last_name": "string",
+  "email": "string",
+  "phone": "string",
+  "job_title": "string",
+  "employee_id": 0,
+  "manager_id": 0,
+  "hire_date": {
+    "year": 2024,
+    "month": 2,
+    "day": 12
+  }
+}
 ```
 
-You view a row added to the **Employees** table as shown below.
+![Try the service](/learn/images/featured-scenarios/build-a-data-service-in-ballerina/try-the-service.gif)
+
+Also, a row will be added to the **Employees** table, as shown below.
 
-![data service output](/images/data-service-guide-output.png)
+![Data service output](/learn/images/featured-scenarios/build-a-data-service-in-ballerina/data-service-output.png)
```

Comment:
Image is not good, as the column names are truncated and also since it is showing data types information is not clear. No need to have data types shown in the diagram.

Reply:
Updated.